src/mobile/src/ui/components/SeedVaultExportComponent.js

```diff
@@ -94,6 +94,30 @@ class SeedVaultExportComponent extends Component {
         seed: '',
     };
 
+    /**
+     * Gets seed vault file path
+     *
+     * @method getPath
+     *
+     * @param {string} prefix
+     *
+     * @returns {string}
+     */
+    static getPath(prefix) {
+        const now = new Date();
+
+        return `${isAndroid ? RNFetchBlob.fs.dirs.DownloadDir : RNFetchBlob.fs.dirs.CacheDir}/${prefix}${
+            // Use local time
+            new Date(
+                now.getTime() - now.getTimezoneOffset() * 60000, // convert minutes to ms because getTime unit is ms
+            )
+                .toISOString()
+                .slice(0, 16)
+                .replace(/[-:]/g, '')
+                .replace('T', '-')
```

Comment:
I think this will be cleaner if we import `moment` from `shared` and simply do `moment().format('YYYYMMDD-HHmm');` to achieve the same result. If you want I can do a PR for that to your branch

Reply:
Thanks @rajivshah3 . I like your suggestion. Done in https://github.com/iotaledger/trinity-wallet/pull/1178/commits/eccf2f496cfb5ff7d238f469276706bc09b5be99